drivers/sensor/vl53l1x/vl53l1_platform_user_config.h

```diff
@@ -0,0 +1,84 @@
+
+/*
+* Copyright (c) 2015 STMicroelectronics International N.V.
+*
+* SPDX-License-Identifier: Apache-2.0
+*
+* Redistribution and use in source and binary forms, with or without
+* modification, are permitted provided that the following conditions are met:
+* 	* Redistributions of source code must retain the above copyright
+* 	  notice, this list of conditions and the following disclaimer.
+* 	* Redistributions in binary form must reproduce the above copyright
+* 	  notice, this list of conditions and the following disclaimer in the
+* 	  documentation and/or other materials provided with the distribution.
+* 	* Neither the name of STMicroelectronics nor the
+* 	  names of its contributors may be used to endorse or promote products
+* 	  derived from this software without specific prior written permission.
+* THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+* ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+* WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE, AND
+* NON-INFRINGEMENT OF INTELLECTUAL PROPERTY RIGHTS ARE DISCLAIMED.
+* IN NO EVENT SHALL STMICROELECTRONICS INTERNATIONAL N.V. BE LIABLE FOR ANY
+* DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+* (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+* LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+* ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+* (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+* SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
```

Comment:
You can remove this now. My point was to sensitize you of this side of things. All things are good now.
You are authorized to delete it.

Reply:
Done, thanks